com.github.zadam.trilium.yml

```diff
@@ -0,0 +1,97 @@
+app-id: com.github.zadam.trilium
+
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+
+command: trilium
+
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
```

Comment:
electron really work with Wayland?

Reply:
Electron v12 and higher should work fine with wayland: https://news.ycombinator.com/item?id=26317942